packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { DocsContext } from '@storybook/addon-docs';
+import dedent from 'dedent';
+import Box from '~components/Box';
+
+export type SandboxProps = {
+  children: string;
+  language?: 'js' | 'ts' | 'jsx' | 'tsx';
+};
+
+function Sandbox({ children, language = 'tsx' }: SandboxProps): JSX.Element {
+  const {
+    // @ts-expect-error globals is available but the typings in storybook are properly defined hence, ignoring it
+    globals: { themeTokenName, colorScheme },
+  } = React.useContext(DocsContext);
+
+  return (
+    <Box>
+      <br />
+      <Sandpack
+        template="react-ts"
+        files={{
+          '/index.tsx': dedent`import { StrictMode } from "react";
+            import * as ReactDOMClient from "react-dom/client";
+            import styled, { createGlobalStyle } from "styled-components";
+  
+            import { BladeProvider, Theme } from "@razorpay/blade/components";
+            import { ${themeTokenName} } from "@razorpay/blade/tokens";
+            import "@fontsource/lato/400.css";
+            import "@fontsource/lato/700.css";
+            
+            import App from "./App";
+  
+            const BackgroundBox = styled.div(
+              ({ theme }: { theme: Theme }) => ({
+                backgroundColor: theme.colors.surface.background.level1.lowContrast,
+                height: '100vh',
+                width: '100vw',
+                padding: '12px'
+              })
+            );
+
+            const GlobalStyles = createGlobalStyle\`
+              body {
+                margin: 0;
+                padding: 0;
+              }
+            \`;
+            
+            const rootElement = document.getElementById("root");
+            
+            if (!rootElement) {
+              throw new Error("root is null");
+            }
+            
+            const root = ReactDOMClient.createRoot(rootElement);
+            
+            root.render(
+              <StrictMode>
+                <BladeProvider themeTokens={${themeTokenName}} colorScheme="${colorScheme}">
+                  <GlobalStyles />
+                  <BackgroundBox>
+                    <App />
+                  </BackgroundBox>
+                </BladeProvider>
+              </StrictMode>
+            );
+            `,
+          [`/App.${language}`]: dedent(children),
+        }}
+        customSetup={{
+          dependencies: {
+            react: '18.0.0',
```

Comment:
what if we keep all the deps same as package.json? we are on React 17. even though 18 is backward compatible not sure if we might come across a bug it'll be difficult to mitigate that it's the version issue

Reply:
makes sense. I'll just hardcode 17 here for now since we don't upgrade react that often. We can update it here as well when we update react.

Reply:
done

Reply:
> ~makes sense. I'll just hardcode 17 here for now since we don't upgrade react that often. We can update it here as well when we update react.~
> 
> Importing package.json and reading from the peerDependencies. This way we'll know if we genuinely break something

yep this is the ideal way 👌